prime-router/src/main/kotlin/fhirengine/utils/FHIRReader.kt

```diff
@@ -0,0 +1,45 @@
+package gov.cdc.prime.router.fhirengine.utils
+
+import gov.cdc.prime.router.ActionLogger
+import gov.cdc.prime.router.InvalidReportMessage
+import org.apache.logging.log4j.kotlin.Logging
+import org.hl7.fhir.r4.model.Bundle
+import java.io.BufferedReader
+import java.io.StringReader
+
+/**
+ * Converts raw FHIR data (ndjson Newline Delimited Json) to Bundle objects.
+ */
+class FHIRReader(private val actionLogger: ActionLogger) : Logging {
```

Comment:
Merge this code with the FHIRTranscoder class. I think the name FHIRReader is fine, but the transcoder class kind of does some of the reading too.

Reply:
Merged FHIRReader into FhirTranscoder